openmetadata-ui/src/main/resources/ui/src/pages/teams/index.test.tsx

```diff
@@ -32,6 +34,10 @@ jest.mock('../../auth-provider/AuthProvider', () => {
   };
 });
 
+jest.mock('@fortawesome/react-fontawesome', () => ({
```

Comment:
Can we mock this module globally in Jest config? Do not need to mock it on every file then

Reply:
@darth-coder00, Sure I will make the changes.